quelf.js

```diff
@@ -0,0 +1,110 @@
+$(document).ready(function() {
+	let p1s = 0;
+	let p2s = 0;
+	let counter = 0;
+	$("#penalty").hide()
+	$("#penalty").hide()
+	class Card{
```

Comment:
Good job with the Card class, however I can see two other ones immediately. A player class can exist and so can a game class. The game class holds all the logic for the game. It contains the players and the cards and the board.